wrapper.js

```diff
@@ -99,13 +99,18 @@ function setupMethods (soljson) {
         } else {
           response.pipe(mem);
           response.on('end', function () {
-            cb(null, setupMethods(requireFromString(mem.toString())));
+            var solcContent = mem.toString();
+            cb(null, setupMethods(requireFromString(solcContent)), solcContent);
```

Comment:
The idea is supporting more interesting use cases rather than just requiring the file, such as caching. This method contains the logic of fetching it, so thought it's an appropriate place. Do you think a separate method would be better?